Ago-Dic-2020/salazar-coronado-juan-daniel/practica2/practica2.py

```diff
@@ -0,0 +1,76 @@
+class Estudiante():
+    def __init__(self, nombre, edad, semestre, carrera):
+        self.nombre = nombre
+        self.edad = edad
+        self.semestre = semestre
+        self.carrera = carrera
+
+    def presentacion(self):
+        return f"Soy {self.nombre}, tengo {self.edad} años y estoy en {self.semestre} semestre."
+
+    def get_carrera(self):
+        carreras = {
+            'ISC': "Ing. En sistemas computacionales",
+            'IIS': "Ing. Industrial y de sistemas",
+            'IEC': "Ing. En electronica y comunicaciones",
+            'ITC': "Ingeniero en tecnología y comunicaciones",
+            'LSA': "Lic. En sistemas computacionales y administrativos",
+            'IA': "Ingeniero automotriz"
+        }
+        return "Estudio la carrera de " + carreras.get(self.carrera)
+    
+    def motivacion_promedio(self, promedio):
+        if promedio >= 70 and promedio <= 79:
+            return "necesito esforzarme más"
+        
+        if promedio > 79 and promedio <= 89:
+            return "lo estoy haciendo bien, pero puedo hacerlo mejor"
+        
+        if promedio >= 90 and promedio <= 100:
+            return "voy muy bien, debo seguir así"
+        
+        return "-> promedio fuera de los parametros normales <-"
+
+class ServicioSocialYPracticas(Estudiante):
+    def __init__(self, nombre, edad, semestre, carrera, servicio, practicas):
+        super().__init__(nombre, edad, semestre, carrera)
+        self.servicio = servicio
+        self.practicas = practicas
+        
+    def estado(self):
+        if self.servicio and self.practicas:
+            return "Ya realice el servicio social y las practicas profesionales"
+        
+        if not self.servicio and not self.practicas:
+            return "No he realizado el servicio social ni las practicas profesionales"
+        
+        if self.servicio == True and self.practicas == False:
```

Comment:
Aquí todavía podemos mejorarlo a `if self.servicio and not self.practicas`. Lo mismo para `if` de la parte de abajo